templates/tests/delete-test-templates.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+set -euo pipefail
+
+# Default to user HOME directory if not specified
+FOLDER_PATH=$HOME
+# Default to 'main' if not specified
+BRANCH_NAME="main"
```

Comment:
NIT: Since we don't use the `BRANCH_NAME` or `LOCATION` in this script, can we remove them (and the remove the stuff that lets you set them based on command line arguments)